utils/zerovec/derive/src/make_varule.rs

```diff
@@ -414,7 +414,9 @@ fn make_encode_impl(
             let ty = &field.field.ty;
             let accessor = &field.accessor;
             quote!(
-                #[allow(clippy::indexing_slicing)] // generate_per_field_offsets produces valid indices
+                // generate_per_field_offsets produces valid indices,
+                // and we don't care about panics in Encode impls
```

Comment:
Well, human-readable deserialize for some VarULE types involves using EncodeAsVarULE.

Do we care about no-panic in JSON runtime mode?

Reply:
No, we already have panics in Encode impls